sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +557,141 @@ public async Task ValidateMergeCertificate()
             CollectionAssert.AreEqual(mergedServerCertificate.Cer, completedServerCertificate.Cer);
         }
 
-        // Backup Restore
-        // GetCertificates
-        // GetCertificatsIncludePending
-        // GetCertificateVersions
-        // GetDeletedCertificates
-        // GetUpdatePolicy
-        // IssuerCrud
-        // ContactsCrud
+        [Test]
+        public async Task VerifyGetIssuer()
+        {
+            string issuerName = WellKnownIssuerNames.Self;
+
+            // Will update the name once get correct provider name.
+            string providerName = "testProvider";
```

Comment:
The provider need register with TenantID and Region, Is there have a document can let me know work on this ?

Reply:
For self-signed certs, you can just set this to something like "CN=Azure SDK" like I did in another test or two.

Reply:
Thanks for your kindly suggestion.
I checked that tests your mentioned found it is CertificatePolicy set subject "CN=Azure SDK" while here is CertificateIssuer's provider. And I tried use such format still not work.

Reply:
What error do you get?

Reply:
Thanks in advance, error as below:
Azure.RequestFailedException: 'Service request failed.
Status: 400 (Bad Request)

Content:
{"error":{"code":"BadParameter","message":"This name is not allowed. Please select a different name."}}

Headers:
Cache-Control: no-cache
Pragma: no-cache
Server: Microsoft-IIS/10.0
x-ms-keyvault-region: eastus
x-ms-request-id: REDACTED
x-ms-keyvault-service-version: 1.1.0.898
x-ms-keyvault-network-info: addr=167.220.233.249;act_addr_fam=InterNetwork;
X-AspNet-Version: REDACTED
X-Powered-By: REDACTED
Strict-Transport-Security: REDACTED
X-Content-Type-Options: REDACTED
Date: Wed, 25 Mar 2020 02:54:04 GMT
Content-Length: 103
Content-Type: application/json; charset=utf-8
Expires: -1

Reply:
@heaths